workers/go/throughputstress/workflow.go

```diff
@@ -146,14 +145,14 @@ func ThroughputStressWorkflow(ctx workflow.Context, params *throughputstress.Wor
 					return nil
 				}
 
-				var sleepInputs []*kitchensink.ExecuteActivityAction
-				genSleepInputs := workflow.SideEffect(ctx, func(ctx workflow.Context) any {
-					return params.SleepActivities.Sample()
+				generateSeed := workflow.SideEffect(ctx, func(ctx workflow.Context) any {
+					return time.Now().UnixNano()
 				})
-				genSleepInputs.Get(&sleepInputs)
+				var random int64
+				generateSeed.Get(&random)
```

Comment:
Generating and using a seed here.

Reply:
you can use workflow.Now().UnixNano() instead of a sideeffect